pkg/plugin/v3/plugin.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v3
+
+import (
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+)
+
+const (
+	pluginName    = "go" + plugin.DefaultNameQualifier
+	pluginVersion = "v3.0.0"
```

Comment:
Should we add the migration guide as well when a new version is bumped? Should we define it as a role?

Reply:
For major versions, yes. Also:
```suggestion
	pluginVersion = "v3.0"
```